common/src/main/scala/hmda/publication/KafkaUtils.scala

```diff
@@ -74,4 +97,33 @@ object KafkaUtils {
 
   }
 
+  def consumeInstitutionRecords(topic: String,
+                                f: Future[Done],
+                                parallelism: Int)(
+      implicit system: ActorSystem,
+      materializer: ActorMaterializer,
+      ec: ExecutionContext) = {
+
+    val config = system.settings.config.getConfig("akka.kafka.consumer")
+
+    val consumerSettings: ConsumerSettings[String, InstitutionKafkaEvent] =
+      ConsumerSettings(config,
+                       new StringDeserializer,
+                       new InstitutionKafkaEventsDeserializer)
+        .withBootstrapServers(kafkaHosts)
+        .withGroupId(UUID.randomUUID().toString)
```

Comment:
You need to have a stable `groupId` here. Otherwise when the process restarts the new consumer will not respect the offset